saveenv.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# change to the proper directory
+cd $(dirname $0)
+
+echo '#!/bin/bash' > aks_baseline.env
+echo '' >> aks_baseline.env
+
+IFS=$'\n'
+
+for var in $(env | grep -E '_AKS_BASELINE' | sort | sed "s/=/='/")
+do
+  echo "export ${var}'" >> aks_baseline.env
+done
```

Comment:
```suggestion
env | sed -n "s/\(_AKS_BASELINE=\)\(.*\)/\1'\2'/p" | sort >> aks_baseline.env
```
as a simple one-liner